app.json

```diff
@@ -13,87 +24,29 @@
   ],
   "env": {
     "BROKER_URL": {
-      "required": true
-    },
-    "BUG_COMMENTER_API_KEY": {
-      "required": true
-    },
-    "BUG_FILER_API_KEY": {
-      "required": true
-    },
-    "CLOUDAMQP_URL": {
-      "required": true
+      "value": "$CLOUDAMQP_URL"
     },
     "DATABASE_URL": {
-      "required": true
-    },
-    "GITHUB_CLIENT_ID": {
-      "required": true
-    },
-    "GITHUB_CLIENT_SECRET": {
-      "required": true
-    },
-    "HEROKU_APP_ID": {
-      "required": true
+      "value": "$JAWSDB_URL"
     },
     "HEROKU_APP_NAME": {
       "required": true
     },
-    "HEROKU_RELEASE_CREATED_AT": {
-      "required": true
-    },
-    "HEROKU_RELEASE_VERSION": {
-      "required": true
-    },
-    "HEROKU_SKIP_NODE_PLUGIN": {
-      "required": true
+    "PULSE_URL": "amqp://treeherder-shared-pulse-user:mozilla123@pulse.mozilla.org:5671/?ssl=true",
+    "PULSE_RESULSETS_QUEUE_NAME": {
+      "generator": "secret"
     },
-    "HEROKU_SLUG_COMMIT": {
-      "required": true
-    },
-    "HEROKU_SLUG_DESCRIPTION": {
-      "required": true
-    },
-    "NEW_RELIC_API_KEY": {
-      "required": true
-    },
-    "NEW_RELIC_APP_NAME": {
-      "required": true
-    },
-    "NEW_RELIC_CONFIG_FILE": {
-      "required": true
-    },
-    "NEW_RELIC_LICENSE_KEY": {
-      "required": true
-    },
-    "PULSE_PUSH_SOURCES": {
-      "required": true
-    },
-    "PULSE_TASK_SOURCES": {
-      "required": true
-    },
-    "REDIS_URL": {
-      "required": true
-    },
-    "SITE_URL": {
-      "required": true
+    "PULSE_TASKS_QUEUE_NAME": {
+      "generator": "secret"
     },
     "TREEHERDER_DJANGO_SECRET_KEY": {
-      "required": true
+      "value": "secret-key-of-at-least-50-characters-to-pass-check-deploy"
```

Comment:
No `"generator": "secret"` here? Not long enough?

Reply:
Addressed! 64 chars so we're good :)